tests/manual/stripe-oxxo.md

```diff
@@ -0,0 +1,54 @@
+## Stripe OXXO
+
+### Set Country to Mexico on your Stripe account Dashboard
+
+### Go to WooCommerce > Settings > Payments > Stripe OXXO
+
+#### Click Enable/Disable checkbox
+
+#### Click Save changes button at the bottom
+
+I see **Your settings have been saved** notice.
+
+Be sure to also copy the webhook endpoint provided on the settings page and add it to your Stripe Dashboard API/Webhook setting.
+
+### Go to WooCommerce > Settings > General
+
+#### Set currency to Mexican Peso ($)
+
+#### Set country to Mexico
```

Comment:
Looks like changing the store country isn't required for OXXO nor Boleto? They both work for me having the United States as the store base.

That aside, what do you think about mentioning testing both the UPE and non-UPE versions in these instructions?